src/config/contributors.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
This PR should be limited to the SQL directories only. Changes related to the contributors PR should be removed.

Reply:
Oh, yes. I was having some issues switching between branches, and I think I did some stuff in the wrong order. I think I figured out what I need to do for the future. As soon as I figure out how to remove the changes from the contributors PR, I will do so!

Reply:
You should be able to straight up delete the files from within this branch and make a new commit that logs those deletions. They'll still exist in the other branch.

I think what might have happened was that you branched from that branch, which included all of its changes. Tip: I always sync/branch from master to ensure a clean branch.

Reply:
`contributors.json` is still showing up in the PR. Can you delete it? Then this should be good to go.